### DISCUSSION TITLE
Yi Dome X

### DISCUSSION THREAD

**Discussion by pitripit** [roleoroleo/yi-hack-Allwinner-v2#183] [General]
Hi,

i have a yi Dome X (BFUS Y29 C4N), which firmware should i use?

**Comment by roleoroleo** [comment]
It depends on the fw version.

**Comment by pitripit** [comment]
firmware ver: 8.1.0.0A_202001211401

**Reply by roleoroleo** [reply]
Try yi-hack-Allwinner (not v2).
Suffix y30qa.

**Comment by pitripit** [comment]
Sorry but where can I find the firmware? On the Release page (
https://github.com/roleoroleo/yi-hack-Allwinner/releases) there are only
y20ga or y25ga.

Can I try to build it from source?



Il giorno dom 6 giu 2021 alle ore 15:01 roleo ***@***.***> ha
scritto:

**Reply by roleoroleo** [reply]
There is no official release at the moment.
Check this issue:
https://github.com/roleoroleo/yi-hack-Allwinner/issues/298

**Comment by pitripit** [comment]
It works. Thanks

Il giorno lun 7 giu 2021 alle ore 21:11 roleo ***@***.***> ha
scritto:
